patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
+index 5d9d58411f2fad9d5da703f964d269b4a7c2b205..0879d6f82803e7da963b029eb1ff6c190ab02b8a 100644
+--- a/src/main/java/net/minecraft/server/WeightedList.java
++++ b/src/main/java/net/minecraft/server/WeightedList.java
+@@ -50,7 +50,9 @@ public class WeightedList<U> {
+     public WeightedList<U> a(Random random) {
+         // Paper start - make concurrent safe, work off a clone of the list
+         List<WeightedList.a<U>> list = isUnsafe ? new java.util.ArrayList<WeightedList.a<U>>(this.list) : this.list;
+-        list.forEach((weightedlist_a) -> weightedlist_a.a(random.nextFloat()));
++        for (a<U> weightedlist_a : list) {
++            weightedlist_a.a(random.nextFloat());
++        }
+         list.sort(Comparator.comparingDouble(a::c));
+         return isUnsafe ? new WeightedList<>(list, isUnsafe) : this;
+         // Paper end
+diff --git a/src/main/java/net/pl3x/purpur/command/PurpurCommand.java b/src/main/java/net/pl3x/purpur/command/PurpurCommand.java
+index 4904be939c7a4b1d1583fd7b6232c930b79caba6..9ccc8601b1354436001167518df85d28e6cda5cb 100644
+--- a/src/main/java/net/pl3x/purpur/command/PurpurCommand.java
++++ b/src/main/java/net/pl3x/purpur/command/PurpurCommand.java
+@@ -9,6 +9,8 @@ import org.bukkit.command.Command;
+ import org.bukkit.command.CommandSender;
+ 
+ import java.io.File;
++import java.util.ArrayList;
++import java.util.Arrays;
+ import java.util.Collections;
+ import java.util.List;
+ import java.util.stream.Collectors;
+@@ -25,9 +27,13 @@ public class PurpurCommand extends Command {
+     @Override
+     public List<String> tabComplete(CommandSender sender, String alias, String[] args, Location location) throws IllegalArgumentException {
+         if (args.length == 1) {
+-            return Stream.of("reload", "version")
+-                    .filter(arg -> arg.startsWith(args[0].toLowerCase()))
+-                    .collect(Collectors.toList());
++            List<String> list = new ArrayList<>();
++            for (String arg : Arrays.asList("reload", "version")) {
++                if (arg.startsWith(args[0].toLowerCase())) {
++                    list.add(arg);
++                }
++            }
++            return list;
+         }
+         return Collections.emptyList();
+     }
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index adf918fd757fe3147f897de3ade64a9adf1d3203..3e175ba71086ee587e86a1b732ea3297772e6b05 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -925,7 +925,9 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
+     @Override
+     public Set<NamespacedKey> getDiscoveredRecipes() {
+         ImmutableSet.Builder<NamespacedKey> bukkitRecipeKeys = ImmutableSet.builder();
+-        getHandle().B().a.forEach(key -> bukkitRecipeKeys.add(CraftNamespacedKey.fromMinecraft(key)));
```

Comment:
Not a stream.

https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/lang/Iterable.java#l72